test/test_site/testUtil/diffHtml.js

```diff
@@ -89,7 +89,7 @@ const diffHtml = (expected, actual) => {
       insidePath = true;
     }
 
-    if (isDiff(part) && !insidePath) {
+    if (isDiff(part) && !insidePath && !isPathSeparatorDiff(part.value)) {
```

Comment:
The change here allows for this situation to pass:
`expected.html`
```html
<div> \ </div>
```

`actual.html`
```html
<div> / <div>
```